modules/renter/negotiate.go

```diff
@@ -110,7 +110,7 @@ func (r *Renter) negotiateContract(host modules.HostSettings, up modules.FileUpl
 	terms := modules.ContractTerms{
 		FileSize:      filesize,
 		Duration:      up.Duration,
-		DurationStart: height - 1,
+		DurationStart: height - 3,
```

Comment:
There's an off-by-one on the network, and there were a significant number of errors being returned about 'DurationStart is in the future', which means that renters and hosts weren't always perfectly synchronized.

This increases the tolerance for de-synchronization slightly.